aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -1,24 +1,14 @@
 import { GuideSection, VideoSection } from '../../examples/foundation';
 
-At HPE, we believe in being a force for good. We solve for humanity with humanity.
-We use technology to make the world better. We are unconditionally inclusive.
-**We believe that being accessible is a requirement, not an option**.
+At HPE, we believe in being a force for good. We solve for humanity with humanity. We use technology to make the world better. We are unconditionally inclusive. **We believe that being accessible is a requirement, not an option**.
 
-<blockquote>
-  "The power of the Web is in its universality. Access by everyone, regardless
-  of disability, is an essential aspect."
-</blockquote>
+Everyone has the right to fully access and take advantage of the web. Web accessibility is focused on making content usable for the widest set of people, including those with disabilities. 
 
-<blockquote fill size="large">
-  &mdash; Tim Berners-Lee, W3C Director and inventor of the World Wide Web
-</blockquote>
+Creating accessible applications means providing experiences that are easily usable by those who rely on keyboard navigation or screen readers, who are colorblind, or who are experiencing some other permanent, temporary, or situational disability.
 
-## Built into all Design System tools
+Accessibility covers a spectrum of disabilities, including but not restricted to blindness, low vision, hearing and cognitive impairments and situational disabilities. From using glasses to aid with sight to a temporary broken bone to carpal tunnel, [about 25% of the United States population](https://www.cdc.gov/ncbddd/disabilityandhealth/infographic-disability-impacts-all.html) lives with some type of disability, permanent or not.
```

Comment:
Thanks for adding. The resource's infographic is great. Love the breakdown of disabilities by type.

Just a side-note to file away for future. Since HPE is a global company and we have a global audience, it'd be great if we could find a reference which is more global in scope. We'll merge in this work, but it'd be good thing to try in future iterations.

Reply:
Or even a mention that this proportion holds true throughout the world.